src/Sylius/Behat/Context/Ui/Shop/ProductContext.php

```diff
@@ -111,4 +141,53 @@ public function iShouldSeeTheProductAttributeWithValue($attributeName, $Attribut
             sprintf('Product should have attribute %s with value %s, but it does not.', $attributeName, $AttributeValue)
         );
     }
+
+    /**
+     * @Given /^I want to see products in (channel "([^"]*)")$/
```

Comment:
Ah, you've already figured this out - awesome! :) I'd just change this step to something like: ``I am browsing channel ...`` or something similar? To make it a generic step for switching channels. Maybe it deserves it's own Context to be reusable in all shop scenarios? We will have a lot of steps like that.